src/main/java/org/wmn4j/notation/elements/Key.java

```diff
@@ -10,7 +10,39 @@
  * @author Otso Björklund
  */
 public enum Key {
-	C_MAJOR, DFLAT_MAJOR, D_MAJOR, EFLAT_MAJOR, E_MAJOR, F_MAJOR, FSHARP_MAJOR, G_MAJOR, AFLAT_MAJOR, A_MAJOR, BFLAT_MAJOR, B_MAJOR,
+	C_MAJOR			(KeySignatures.CMAJ_AMIN),
```

Comment:
These could be done without whitespace between the enum name and the parentheses.

Reply:
Done :+1: